tests/api/test_list_messages.py

```diff
@@ -373,7 +373,7 @@ async def fetch_messages_with_pagination(
 ):
     return await api_client.get(
         MESSAGES_URI,
-        params={"page": page, "pagination": pagination, "sort_order": sort_order},
+        params={"page": page, "pagination": pagination, "sortOrder": sort_order},
```

Comment:
Would this break backward compatibility with current software using `sort_order` ? How can we ensure it does not ?

Reply:
Yes, better double check before to merge this PR. The missing aliases were an irregularity, since camelCase is used for key names at this endpoint. I also checked the JS SDK, sorting is not part yet of param queries, so all good there.

Reply:
I just double checked, for PyAleph, Python SDK and JS SDK, it's ok. Then I don't know if/which other repos are using those endpoints.